src/components/RSVPform.js

```diff
@@ -0,0 +1,164 @@
+import {addUser} from "../firebase/db";
+import {Form, Button, Col, FormControl, InputGroup} from "react-bootstrap";
+import React, {useState} from "react";
+
+const RSVPform = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [rsvp, setRsvp] = useState("");
+  const [plusOne, setPlusOne] = useState(false);
+  const [plusFName, setPlusFName] = useState("");
+  const [plusLName, setPlusLName] = useState("");
+  const [allergies, setAllergies] = useState("");
+  const [guestEmail, setEmail] = useState("");
+
+  const clearFields = () => {
+    setFirstName("");
+    setLastName("");
+    setRsvp("");
+    setPlusOne(false);
+    setPlusFName("");
+    setPlusLName("");
+    setAllergies("");
+    setEmail("");
+  };
+
+  const handleSubmit = (e) => {
+    const guest = {
+      firstName: firstName,
+      lastName: lastName,
+      uid: new Date().getTime(),
```

Comment:
So this would be a good place to use a UUID. While unlikely with only a few people using the app at a time. Timestamps will collide at scale.